taxonomies/categories.txt

```diff
@@ -20908,6 +20980,26 @@ wikipedia:en:https://en.wikipedia.org/wiki/Crouton
 # ingredient/fr:croûtons has 107 products in 5 languages @2019-04-21
 # category/croutons has 166 products @2019-05-19
 # usage:en:croutons (wheat flour, vegetable fat, edible salt, maltodextrin, grape sugar, onions, herbs, yeast extract, dehydrated garlic, flavoring, yeast, spice extract)
+ciqual_food_code:en:7430
+ciqual_food_name:en:Croutons
+
+<en:Breads
+en:Croutons
```

Comment:
there's already a "en:Croutons" category above, we can't have a category with the same name. 
I have no idea what a "Croûtons à tartiner" is though :)     Les pains suédois ?

Reply:
mmh…two different ids though